genestack_client/settings/genestack_user.py

```diff
@@ -97,3 +97,14 @@ def __interactive_login(self, connection):
             except GenestackAuthenticationException:
                 message = 'Your username or password was incorrect for %s. Please try again.' % self.host
 
+    def __eq__(self, other):
+        return (
+            isinstance(other, User) and
+            self.alias == other.alias and
```

Comment:
Do we need to check that aliases match?  I mean, it might be the same user with different aliases.

Reply:
This class is slightly inner, and not planned to use by API users a lot.  I prefer to leave it as comparing User class instances, to use it in the tests.  If API user create two aliases to same genestack user, it is  not a bug, it it decision of grownup independent person.